configparser.go

```diff
@@ -18,13 +18,14 @@
 package main
 
 import (
-	"github.com/ifwe/goconf/conf"
-	. "github.com/uniqush/log"
-	. "github.com/uniqush/uniqush-push/db"
-	. "github.com/uniqush/uniqush-push/push"
 	"io"
 	"os"
 	"strings"
+
+	conf "github.com/pelletier/go-toml"
```

Comment:
toml is nice. But we do not want to break our API, not until we bump to version 2.0